src/style.css

```diff
@@ -1 +1,45 @@
-/* add styles here */
+body {
+  margin: 0;
+  box-sizing: border-box;
```

Comment:
You should avoid using the `box-sizing` property on the `body` tag. This property should be applied to all elements on the page. This is done to ensure that the `padding` and `border` of an element no longer increase its width. You can do this by using the `*` selector, but remember, it's not recommended to use `*` selector for other styles. Here is how you can do it: css
*, *::before, *::after {
  box-sizing: border-box;
}
```